les1.js

```diff
@@ -0,0 +1,8 @@
+const fn1 = (sayHello) => {
```

Comment:
bad naming. your function accept some name, but you named your param like a function